tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -325,9 +310,21 @@ const reducer = createReducer(
       state.pinnedCardToOriginal,
       state.cardStepIndex
     );
+
+    const hydratedSmoothing = hydratedState.metrics.smoothing;
+    const newSmoothing =
+      Number.isFinite(hydratedSmoothing) && hydratedSmoothing !== null
+        ? Math.max(0, Math.min(0.999, hydratedSmoothing))
```

Comment:
Can we pull out the same hardcoded 0.999 value from 'webapp/metrics/views/right_pane/settings_view_component.ts' and reference `MAX_SMOOTHING_VALUE` here?